instruction.json

```diff
@@ -0,0 +1,24 @@
+{"instructions": 
+  {
+    "projection": 2193,
+	"grid_params": {
+      "resolution": 10
+    },
+    "data_paths": {
+      "catchment_boundary": "C:\\Users\\pearsonra\\Documents\\data\\Wakanae\\Small_test_site\\initial_data\\boundary\\boundary.shp",
```

Comment:
Could these be in a relative path format, such as in the test example?

Reply:
They could be - I was thinking of removing this from version control once the test is created and working. I will leave for now and remove prior to the end of this pr.